CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md).
 
 ## [Unreleased]
+### Fixed
+- Remove unsupported Ruby version 2.1
```

Comment:
we should remove version support for `< 2.3.0`

Reply:
:+1: